AppCenter/AppCenterTests/MSAuthTokenContextTests.m

```diff
@@ -472,14 +466,16 @@ - (void)testRefreshAuthTokenForAccountId {
 - (void)testFinishDoesNotResetTokenIfPrevented {
 
   // If
-  [self.sut preventResetAuthTokenAfterStart];
-  OCMReject([self.sut setAuthToken:OCMOCK_ANY withAccountId:OCMOCK_ANY expiresOn:OCMOCK_ANY]);
+  id sut = OCMPartialMock(self.sut);
+  [sut preventResetAuthTokenAfterStart];
+  OCMReject([sut setAuthToken:OCMOCK_ANY withAccountId:OCMOCK_ANY expiresOn:OCMOCK_ANY]);
 
   // When
-  [self.sut finishInitialize];
+  [sut finishInitialize];
 
   // Then
-  OCMVerify(self.sut);
+  OCMVerifyAll(sut);
```

Comment:
Do we really need the `OCMVerifyAll` the REJECT should crash the tests if the method is called.

Reply:
yep.

> In fail-fast mode an exception might not cause the test to fail.

See 9.2 in http://ocmock.org/reference/